spec/card_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'poker_hands/card'
+require 'spec_helper'
+
+
+RSpec.describe PokerHands::Card do
+  subject { PokerHands::Card.new(5, 'C') }
+
+  context 'Card object has expected rank and suit' do
```

Comment:
Since we don't have the inverse case I wouldn't use a context, I would go directly to our `it` blocks.